### PR TITLE
Aims 408 add deaccessioned status

### PR DESCRIPTION
**Pull request by afstanton**
Code seems to work, but I could use some help making the test work.

### CODE REVIEW

**Comment by afstanton** [comment]
I agree that the tests need to be fixed, but I could use some help with that, please - I've tried to follow established patterns in testing search results with Solr, but nothing seems to work properly.

**Comment by afstanton** [comment]
I agree that the tests need to be fixed, but I could use some help with that, please - I've tried to follow established patterns in testing search results with Solr, but nothing seems to work properly.

**Comment by afstanton** [comment]
I'm still stuck on this - help would be appreciated.

**Comment by rfox2** [comment]
By the way, my approval alone is OK for this one - there's no way Justin will have time to do code reviews for the next couple of weeks.

**Comment by rfox2** [comment]
Aaron: can you resolve the conflicts on this? I'll look at it ASAP after that. Thanks!

File: db/migrate/20170531235744_reindex_items.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class ReindexItems < ActiveRecord::Migration
```
Comment (jgondron) [inline db/migrate/20170531235744_reindex_items.rb]: This was already communicated in an email, but just noting it here for anyone else who looks at this PR. This is going to be moved out of a migration and into a rake task that we can perform post deploy.

**Comment by afstanton** [comment]
Migration has been removed. No new rake task seems to be needed, I believe that running `bundle exec rake sunspot:reindex` should handle the required reindex.

**Pull request merged**